pages/_document.tsx

```diff
@@ -4,6 +4,22 @@ export default function Document() {
   return (
     <Html lang="en">
       <Head>
+        {/* Structured Metadata using JSON-LD */}
+        <script
```

Comment:
What is this for?

Reply:
I have provided the explanatory comment.